src/site/content/pl/pl.11tydata.js

```diff
@@ -1,3 +1,7 @@
+const livePaths = require('../../_filters/live-paths');
```

Comment:
It seems like it'd be good for us to remove the paths data from this file in the future so we don't have to update every language if we change the paths.